### ISSUE TITLE
Slow load of the gif when first loading.

### ISSUE THREAD

**Issue by themasterfra** [RafaelBarbosatec/gif_view#18]
When the gif is first loadead it takes 3 to 5 seconds to show, even with low size gif.

**Comment by RafaelBarbosatec** [comment]
Hi @themasterfra !
Do you can provide more informations? I'm sorry. I can't reproduce this.

**Comment by themasterfra** [comment]
When the gif is first loaded in a dropdown menù, it takes 1-2 seconds to
load in the cache.
To solve it i moved the load in the cache at the start up of the app.

Il Mar 30 Gen 2024, 21:24 Rafael Almeida Barbosa ***@***.***>
ha scritto:

**Comment by harshSalescode** [comment]
The Gif is taking 2-5 seconds to load fir the first time. Also, at every screen update the GIF is updating and there is a blink effect with this.

**Comment by Jatin5120** [comment]
The problem is with the gif image caching, probably we can add a way to precache the gif.
I am using a GIF as the loader. It doesn't show up for the first 1-2 times.

**Comment by themasterfra** [comment]
Yes, as i said i moved the load of the gif at the start up of the app, and works well. 
So a precache is needed.

**Comment by Jatin5120** [comment]
@themasterfra  How did you move the load of gif to start of the app? Mind helping with the code?

**Comment by RafaelBarbosatec** [comment]
Hi everyone!
If needed, prefetch is available in the latest version
